src/genegraph/source/graphql/gene.clj

```diff
@@ -72,59 +73,30 @@
 
 (defn hgnc-id [context args value]
   (->> (q/ld-> value [:owl/same-as])
-       (filter #(= (str (q/ld1-> % [:dc/source])) "https://www.genenames.org"))
+       (filter #(= (str (ld1-> % [:dc/source])) "https://www.genenames.org"))
        first
        str))
 
 (defn curations [context args value]
-  (let [actionability (q/ld-> value [[:sepio/is-about-gene :<] [:sepio/is-about-condition :<]])]
+  (let [actionability (ld-> value [[:sepio/is-about-gene :<] [:sepio/is-about-condition :<]])]
     (map #(tag-with-type % :actionability_curation)) actionability))
 
 (defn conditions [context args value]
   (get value [:sepio/is-about-gene :<]))
 
 ;; TODO check for type (hopefully before structurally necessary)
 (defn actionability-curations [context args value]
-  (q/ld-> value [[:sepio/is-about-gene :<] [:sepio/is-about-condition :<]]))
-
+  (ld-> value [[:sepio/is-about-gene :<] [:sepio/is-about-condition :<]]))
 
 (defn dosage-curation [context args value]
   (let [query (create-query [:project ['dosage_report] (cons :bgp has-dosage-bgp)])]
     (first (query {::q/params {:limit 1} :gene value}))))
 
-(defn hgnc-symbol [context args value]
-  (q/ld-> value []))
-
-(defn gene-type [context args value]
-  (q/ld-> value []))
-
-(defn locus-type [context args value]
-  (q/ld-> value []))
-
 (defn previous-symbols [context args value]
-  (q/ld-> value []))
-
-(defn alias-symbols [context args value]
-  (q/ld-> value []))
-
-(defn chromo-loc [context args value]
- (q/ld-> value []))
+  (str/join ", " (ld-> value [:skos/hidden-label])))
 
-(defn function [context args value]
-  (q/ld-> value []))
-
-(defn coordinates [context args value]
-  (q/ld-> value []))
-
-(defn build [context args value]
-  (q/ld-> value []))
-
-(defn chromosome [context args value]
-  (q/ld-> value []))
-
-(defn start-pos [context args value]
-  (q/ld-> value []))
-
-(defn end-pos [context args value]
-  (q/ld-> value []))
+(defn alternative-label [context args value]
```

Comment:
I think :skos/alternate-label isn't a property, you mean :skos/alternative-label? (which I think corresponds with the gene name)

Reply:
Reverted back to using the resource version so this is removed.